### ISSUE TITLE
Date not fully localized

### ISSUE THREAD

**Issue by Quent-in** [diaspora/diaspora#7527]
I noticed that the name of the months on the notification page are not localized.
Like is this picture, they remain in English:
![](https://framapic.org/SKrKM1myeaqb/o1B8Wn5tY357)

**Comment by Flaburgan** [comment]
The translation you are using is not complete. The months are correctly translated with the locale set to french for example. Please complete the Occitan translations on http://webtranslateit.com

**Comment by SuperTux88** [comment]
The month names are not on webtranslateit, we use `I18n.localize` for that which uses the translations from `rails-i18n`, and Occitan is not translated there: https://github.com/svenfuchs/rails-i18n/tree/master/rails/locale

I think we could add `month_names` to our locales, but then it would pop up on all languages on webtranslateit, even when it's already translated in `rails-i18n`. But you can maybe contribute the Occitan translation to [rails-i18n](https://github.com/svenfuchs/rails-i18n#how-to-contribute), so every rails project will profit from it.

**Comment by Quent-in** [comment]
I'll complete rails-i18n and submit a PR here too in case they don't accept it before diaspora* next release.
Thanks for your help, I'll try to do it before Monday.

**Comment by Quent-in** [comment]
Hello
I put the file here: https://github.com/Quent-in/rails-i18n/blob/708018b02f022996e12d2c5ebc632503efb96ada/rails/locale/oc.yml
if you could add it to the railsi18n folder of diaspora*

**Comment by Quent-in** [comment]
Any news about it?
Best regards

**Comment by SuperTux88** [comment]
Sorry I didn't answer here earlier. We decided not to manually copy your changes to the translation files, because it would be manual work that would be reverted everytime I pull translations from WebTranslateIt and also we couldn't really test it, and travis was red at the time I looked at your PR.

But since your PR is now merged we can update `rails-i18n` as soon as there is a new release for it.
